_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+Have you ever been sandwiched between sharing a URL to NewPipe, and a little "Unsupported URL" toast? Well, those URLs are still unsupported, but now the error message is great! It allows you to reshare the URL or just open it in a web-browser instead. All this, thanks to [@Stypox](https://github.com/Stypox) and [@webber-naut](https://github.com/webber-naut) for bringing this to our attention in the first place. 
 
-Talking about sharing, when you share a live stream with someone, NewPipe doesn't add a timestamp to the URL any more. This was done by [@nmurali94](https://github.com/nmurali94). Thanks.
+Talking about sharing, when you share a live stream with someone, NewPipe doesn't add a timestamp to the URL any more. This was done by [@nmurali94](https://github.com/nmurali94). Thanks :)
 
-The header layout has been improved: the alignment on the sides is consistent now and longer headings can fit themselves on two lines instead of just one.
+The header layout has improved: The alignment on the sides is consistent, and longer headings can expand to two lines.
 
-If you ever tried to search something and then lost connection to the internet, or just tried to search while being offline, you would have noticed that NewPipe didn't let you type your search term. Instead it kept on rudely interrupting you with a network error. That is fixed now. You can type all you want, and it will only fail if you actually hit enter or press the search button. Both of these changes were brought to you by [@Stypox](https://github.com/Stypox). Thanks!
+The thing worse than searching while offline, or losing your connection to the Internet when doing it, is that NewPipe didn't let you type your search term.
+Being rudely interrupted by a network error is a thing of the past. Type all you want, and accept defeat after the fact. Both of these changes brought to you by [@Stypox](https://github.com/Stypox). Thanks!
 
-In order to save some of that valuable mobile data, you were already able to turn off video thumbnails and profile pictures for comments. Now even the black placeholders which replaced them are gone and comments can comfortably take up the space left behind. Thanks, [@4D17Y4](https://github.com/4D17Y4)!
+To save valuable mobile data, turning off video thumbnails and profile pictures for comments was already possible. Now even the black placeholders which replaced them are gone, and comments can comfortably take up the space left over. Thanks, [@4D17Y4](https://github.com/4D17Y4)!
 
-[@TobiGr](https://github.com/TobiGr) brought two more improvements into this release: the setting to remember pop-up size and position has a less redundant description, and when you switch to another app, the main player now minimizes to background audio by default to allow for a more seamless transition. Swoosh.
```

Comment:
What? What? What? Oh, at the end. Well, swoosh.

Reply:
xD sorry, what?